explorer/client/src/pages/object-result/ObjectLoaded.tsx

```diff
@@ -223,12 +223,36 @@ function ObjectLoaded({ data }: { data: DataType }) {
                                     />
                                 </div>
                             </div>
+                            {data.data?.tx_digest && (
+                                <div>
+                                    <div>Transaction ID</div>
```

Comment:
We might want to say something like "Last Transaction ID" to give more clarity on what this means.

Extra bonus points for a help tooltip that says "The ID of the last transaction to read or write this object. If this object has not been read or written by any transactions, this is the transaction that created the object".